src/modules/75-cf/pages/onboarding/OnboardingDetailPage.module.scss

```
@@ -5,15 +5,38 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-.mainContent {
+.fullScreenContainer {
   overflow: hidden;
-  height: var(--page-min-height);
+  height: calc(var(--page-min-height) - var(--spacing-huge));
+  margin-left: 250px !important;
+}
+
+.iconContainer {
+  display: flex;
+  justify-content: space-between;
 }
 
 .onboardingContainer {
   overflow-y: scroll;
+  overflow-x: scroll;
```

Since you're specifying both the X and Y as the same thing, you can just use `overflow` instead. Also `scroll` will always show scrollbars, whereas `auto` will only show if they are required.
```suggestion
  overflow: auto;
```

---

Awesome, thanks :)